content/docs/apis.md

```diff
@@ -2,57 +2,196 @@
 title: APIs
 description: Interact with TiKV using the raw key-value API or the transactional key-value API
```

Comment:
```suggestion
description: Interact with TiKV using the Raw Key-Value API or the transactional key-value API
```

Reply:
We may want to unify the API names of Raw Key-Value API and Transactional Key-Value API (initial-capitalized),

Reply:
I'll do little `raw` and little `transactional` then. :)